src/core/client/admin/components/StoryInfoDrawer/Checkbox.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent } from "react";
+import { Control, useController } from "react-hook-form";
+
+import { CheckBox } from "coral-ui/components/v2";
+
+export interface Props {
+  name: string;
+  control: Control;
+  defaultValue: any;
+  children: React.ReactChildren | React.ReactChild;
+}
```

Comment:
Does React Hook Form handle disabling the control during the submitting state for us? 

If not, while we wait for a response during submission do we want to disable the checkbox by passing in an optional `disabled` prop here?